catalog/ui/catalog-ui-search/src/main/webapp/component/search-settings/search-settings.view.js

```diff
@@ -97,6 +102,9 @@ define([
             this.updateSearchSettings();
             user.savePreferences();
             this.$el.trigger('closeDropdown.'+CustomElements.getNamespace());
+        },
```

Comment:
In the cancel we'll want to call `onBeforeShow` again to reset the form.  

A few other changes we should make:
- Update the save to also call onBeforeShow.
- Update onBeforeShow to put the listener for the region into an if statement like so:
```
if (this.options.showSave !== true) {
     this.listenToOnce(this.regionManager, 'before:remove:region', this.save);
}
```